_posts/2020-03-12-tcf2.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Prebid.org Support for TCF v2.0
+description:
+permalink: /blog/tcf2
+---
+
+
+<br>
+
+The IAB's Transparency and Consent Framework (TCF) version 2.0 for enhanced support of GDPR is scheduled to take effect April 1, 2020. This is a major update from TCF version 1.1.
+
+The key changes are:
+- More defined 'purposes'.
+- More flexibility for the legal bases used by vendors.
+- Different in-page JavaScript API.
+
+References
+- [IAB Europe's TCF v2.0 Homepage](https://iabeurope.eu/tcf-2-0/)
+- [TCF Policies](https://iabeurope.eu/tcfdocuments/documents/legal/tcfpolicyFINALv2.pdf)
+- [TCF v2.0 Consent String Format](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md)
+- [Prebid Support for Enforcing TCF v2.0](https://docs.google.com/document/d/1fBRaodKifv1pYsWY3ia-9K96VHUjd8kKvxZlOsozm8E/edit#)
+
+{: .alert.alert-warning :}
+**Consult with your legal counsel before determining how to utilize Prebid features in support of your overall privacy approach.**
+
+The following is a summary of how various Prebid components will support TCF v2.0 and the estimated availability dates:
+
+## Prebid.js
+
+{: .table .table-bordered .table-striped }
+| Phase | Description | Est. Availability |
+| ----- | ----------- | ----------------- |
+| 1 | Support ['deviceAccess' and 'defaultGdprScope'](https://github.com/prebid/Prebid.js/issues/4747) flags. (Publishers will need to parse the TCF string on their own and configure Prebid.js appropriately.) | Mar 18, 2020 |
+| 2 | Update the GDPR ConsentManagement module to [read TCF v2.0 strings](https://github.com/prebid/Prebid.js/issues/4801) and forward to bid adapters. | Mar 18, 2020 |
+| 3 | Optionally parse TCF v2.0 strings and enforce Device Access. | Apr 1, 2020 |
```

Comment:
I'm not sure what it means to "optionally parse" something. What is optional here?